README.md

```diff
@@ -146,6 +146,7 @@ Set up your GitHub Actions workflow with a specific version of your programming
 - [GitHub Actions for Logtalk and Prolog](https://github.com/logtalk-actions)
 - [GitHub Actions for Deno](https://github.com/denolib/setup-deno)
 - [GitHub Actions for Unity](https://github.com/webbertakken/unity-actions)
+- [GitHub Actions for Cocoapods](https://github.com/maxim-lobanov/setup-cocoapods)
```

Comment:
Again, although there are two different configurations, the underlying action is that it sets up cocoapods. Would you mind moving this to utility as well? I will make sure to add a "Mobile" section in the next week and move the appropriate actions there.

Reply:
Fixed, thank you for your feedback